dot/rpc/modules/author_test.go

```diff
@@ -410,28 +480,32 @@ func TestAuthorModule_InsertKey(t *testing.T) {
 				coreAPI:    tt.fields.coreAPI,
 				txStateAPI: tt.fields.txStateAPI,
 			}
-			if err := am.InsertKey(tt.args.r, tt.args.req, tt.args.res); (err != nil) != tt.wantErr {
-				t.Errorf("AuthorModule.InsertKey() error = %v, wantErr %v", err, tt.wantErr)
+			err := am.InsertKey(tt.args.r, tt.args.req, tt.args.res)
+			if tt.wantErr {
+				if assert.NotNil(t, err) {
+					assert.EqualError(t, err, tt.err.Error())
+				}
+			} else {
+				assert.NoError(t, err)
 			}
 		})
 	}
 }
 
 func TestAuthorModule_HasKey(t *testing.T) {
+	kr, err := keystore.NewSr25519Keyring()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Don't panic in tests, use `t.Fatal` or `require.XXX` functions instead

```suggestion
	require.NoError(t, err)
```